pkg/crypto/age.go

```diff
@@ -116,6 +134,10 @@ func SetupAge(client api.Client, emails []string) error {
 		if err != nil {
 			return err
 		}
+		missingEmails := findMissingKeyForEmail(emails, keys)
```

Comment:
findMissingKeyForEmail should just return the set and then call empty below (saves an array allocation and should be constant time)

Reply:
yes but when I want to display emails that have no keys setup I will have this:
```
Some of the users map[test@email.com:true] have no keys setup
```
instead of this message:
```
Some of the users [test@email.com] have no keys setup
```
So in the current version, I don't need to make any conversion